src/main/java/com/iotiq/application/service/CustomerService.java

```diff
@@ -106,9 +106,9 @@ private Customer createCustomer(User currentUser) {
         return customerRepository.save(customer);
     }
 
-    private List<OrderDto> setLastTwoOrders(List<OrderDto> orderDtoList) {
+    private List<OrderDto> getLastTwoOrders(List<OrderDto> orderDtoList) {
         if (orderDtoList == null || orderDtoList.isEmpty()) {
-            return orderDtoList;
+            return Collections.emptyList();
```

Comment:
Actually returning an emptyList is better than returning null. But what I actually meant is to use collectionUtils.isEmpty instead of orderDtoList == null || orderDtoList.isEmpty()